templates/visuals/.api/v1.6.0/schema.capabilities.json

```diff
@@ -97,7 +97,7 @@
                         "type": "object",
                         "description": "condition - Defines conditions for a data mapping (each key needs to be a valid data role)",
                         "patternProperties": {
-                            "^\\w+$": {
+                            "^[\\w\\s-]+$": {
```

Comment:
This pattern allows to use:
- a-zA-Z
- "-" (dash sign)
- spaces
•	0-9

Examples:
- Power BI
- Power_BI1
- Power-BI